locales/messages.en.json

```diff
@@ -103,9 +106,12 @@
   "ExpectedFailOrSkip": "expected 'fail', 'skip', or 'pass' here",
   "ExpectedPortName": "expected a port name here",
   "ExpectedTripletName": "expected a triplet name here",
+  "ExtendedDocumenationAtUrl": "Extendend documentation available at '{url}'.",
   "FailedToProvisionCe": "Failed to provision vcpkg-ce.",
   "FailedToRunToolToDetermineVersion": "Failed to run {path} to determine the {tool_name} version.",
   "FailedToStoreBackToMirror": "failed to store back to mirror:",
+  "FailedToStoreBinaryCache": "Failed to store binary cache '{path}':'{error_msg}'",
+  "FailedVendorAuthentication": "One or more {vendor} credential providers failed to authenticate. See '{url}' for more detailson how to provide credentials.",
```

Comment:
```suggestion
  "FailedVendorAuthentication": "One or more {vendor} credential providers failed to authenticate. See '{url}' for more details on how to provide credentials.",
```

Reply:
Note that the json files shouldn't be edited by hand: the C++ code that created these entries should be fixed and generate-messages rerun.

Reply:
Ah, yes. I've added the comments on the related code changes.